test.sh

```diff
@@ -94,7 +99,7 @@ case ${ACTION} in
   ;;
 "pylint")
   setup_dfp
-  $RUN $PKG install -y "${PYTHON}-pylint"
+  $RUN $PIP install pylint
```

Comment:
We used distro specific version intentionally. Upstream pylint changes too often and causes many false positives we have to handle.

Reply:
> Upstream pylint changes too often and causes many false positives we have to handle.

Would not this be better addressed by pip installing a specific version, rather then relying on the fact that the distro packaging is somewhat behind the latest?

Reply:
It happens that when installing from pip some dependencies were broken, or incompatible dependencies were installed. We would need to freeze pip versions. So for me it's easier to rely on distro installations that resolves dependency issues for me.